packages/framework/src/Resources/views/Common/Inline/Icon/icon.html.twig

```diff
@@ -0,0 +1,8 @@
+{% if type == 'svg-font' %}
+    <i class="svg svg-{{ name }} {{ attr.class|default('')}}"
+        {% if attr.title|default('') %}title ="{{ attr.title }}"{% endif %}
+        {% for key, value in attr.data|default('') %}
```

Comment:
I would definitely set default values in PHP method. Iterating on empty string seems really weird. I am amazed it is working withou throwing any exception 😲 .

Reply:
changed

Reply:
Actually it is not.